client/src/components/forms/Login.js

```diff
@@ -1,16 +1,23 @@
 import React, { useState } from "react";
 import styled from "styled-components";
-import { Link } from "react-router-dom";
+import { Link, useHistory } from "react-router-dom";
 import { postData } from "../../helpers/fetch";
 
-function Login() {
+function Login({ setUserData, userData }) {
+  console.log(setUserData);
   const [input, setInput] = useState({});
+  const history = useHistory();
 
-  const handleSubmit = () => {
-    fetch("/login", {
+  const handleSubmit = async () => {
+    const apiCall = await fetch("/login", {
```

Comment:
I love that you've explored the `fetch` API. I'd also like you to check out other HTTP clients like [axios](https://github.com/axios/axios) and compare their features -- i.e if you haven't already.

Just a thought, you don't necessarily have to switch to axios here unless you want to!